target/linux/ath79/dts/qca9557_mikrotik_routerboard-960pgs.dts

```diff
@@ -0,0 +1,310 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca955x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "mikrotik,routerboard-960pgs", "qca,qca9557";
+	model = "MikroTik RouterBOARD 960PGS";
```

Comment:
maybe adjust to "MikroTik RouterBOARD 960PGS (hEx PoE)" as is done for other RB devices in tree

Reply:
I'll change this in next push